package.json

```diff
@@ -1,37 +1,38 @@
 {
-  "name": "express-authorshaven",
+  "name": "tech-strikers-bn-backend",
   "version": "1.0.0",
-  "description": "A Social platform for the creative at heart",
+  "description": "[![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)",
```

Comment:
I think this should be the project description.
Hound badge has to be added in the readme.